### ISSUE TITLE
pqcrypto-rainbow test fails compilation

### ISSUE THREAD

**Issue by mouse07410** [rustpq/pqcrypto#42]
Relevant (IMHO) error
```
$ cargo test
.  .  .
error: use of deprecated constant `ffi::PQCLEAN_RAINBOWVCIRCUMZENITHAL_CLEAN_CRYPTO_PUBLICKEYBYTES`: Insecure cryptography, do not use in production
    --> pqcrypto-rainbow/src/ffi.rs:1183:40
     |
1183 |             let mut pk_alt = vec![0u8; PQCLEAN_RAINBOWVCIRCUMZENITHAL_CLEAN_CRYPTO_PUBLICKEYBYTES];
     |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
.  .  .
warning: `pqcrypto-rainbow` (lib test) generated 234 warnings (171 duplicates)
error: could not compile `pqcrypto-rainbow` due to 153 previous errors; 234 warnings emitted
warning: build failed, waiting for other jobs to finish...
```

**Comment by thomwiggers** [comment]
You need to enable the `insecure-cryptography` feature. I might need to point this out in more places.

**Comment by mouse07410** [comment]
I did not enable insecure crypto for the build. So, why is it being tested (and then fails)?

I'd suggest that if it's supposed to be disabled - it should be disabled completely: doesn't get compiled or tested, unless a special setting is given.

**Comment by thomwiggers** [comment]
Closed by #43
